### PR TITLE
Language and style edit to improve readability

### PR DESCRIPTION
**Pull request by radhikapc**
Signed-off-by: radhikapc <radhika.dwaraka@gmail.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

 /area blog

> /area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Language and style edit to improve readability.

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/blog/falco-i18n.md [hunk 1]
```diff
@@ -1,107 +1,104 @@
...
 
 
-## Review Process
+## Become a Maintainer
 
-Falco i18n members can review and approve their own PRs. For example, review and approval permissions for English are found in the [OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS) file at the root of [Falco-website]( https://github.com/falcosecurity/falco-website).
+Falco i18n members can review and approve their own PRs. For example, review and approval permissions for English are found in the [OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS) file at the root of the [Falco-website]( https://github.com/falcosecurity/falco-website) repository.
 
-As a contributor, make sure that you  create an OWNER file inside the `content/<lang code>` directory in order to become a maintainer for that localization.
+As a contributor, make sure that you create an OWNER file inside the `content/<lang code>` directory in order to become a maintainer for that localization.
```
Comment (leogr) [inline content/en/blog/falco-i18n.md:104]:
```suggestion
As a contributor, make sure that you create an OWNERS file inside the `content/<lang code>` directory in order to become a maintainer for that localization.
```

File: content/en/blog/falco-i18n.md [hunk 2]
```diff
@@ -1,107 +1,104 @@
...
 
-13. Ensure that you sign your PR with `git commit -s -m "message”`
+13. Once ready create the PR against the master. Make sure you split the effort in smaller units.
 
 
-## Review Process
+## Become a Maintainer
 
-Falco i18n members can review and approve their own PRs. For example, review and approval permissions for English are found in the [OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS) file at the root of [Falco-website]( https://github.com/falcosecurity/falco-website).
+Falco i18n members can review and approve their own PRs. For example, review and approval permissions for English are found in the [OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS) file at the root of the [Falco-website]( https://github.com/falcosecurity/falco-website) repository.
```
Comment (leogr) [inline content/en/blog/falco-i18n.md:102]:
```suggestion
Falco i18n members can review and approve their own PRs. For example, review and approval permissions for English are found in the [OWNERS](https://github.com/falcosecurity/falco-website/blob/master/OWNERS) file at the root of the [falco-website](https://github.com/falcosecurity/falco-website) repository.
```

File: content/en/blog/falco-i18n.md [hunk 3]
```diff
@@ -1,107 +1,104 @@
...
 
-12. Once ready create a PR against the master. Make sure you split the effort in smaller units.
+12. Ensure that you sign your commit with `git commit -s -m "message”`.
 
-13. Ensure that you sign your PR with `git commit -s -m "message”`
+13. Once ready create the PR against the master. Make sure you split the effort in smaller units.
 
 
-## Review Process
+## Become a Maintainer
```
Comment (leogr) [inline content/en/blog/falco-i18n.md:100]: Perhaps, it worth mentioning https://github.com/falcosecurity/.github/blob/master/GOVERNANCE.md ?

Comment (radhikapc) [reply content/en/blog/falco-i18n.md:100]: will add the link.

**Review by leogr — APPROVED** [review]
Thank you!

**Pull request merged**